docs/user/dashboard/url-drilldown.asciidoc

```diff
@@ -239,6 +239,23 @@ Tip: Consider using <<helpers, date>> helper for date formatting.
 
 |===
 
+[float]
+[[url-drilldown-supported-panels]]
+==== Supported panels
```

Comment:
Suggest moving this section up higher in the doc, either before or after the "Try it" section.